packages/sdk/src/get-generators/utils/check-feature-flags/forbiddenTransactionsWithProxyFlagMessage.ts

```diff
@@ -0,0 +1,20 @@
+import chalk from 'chalk'
+import { highlightDatamodel } from '../../../highlight/highlight'
+import { link } from '../../../link'
+
+export const forbiddenTransactionsWithProxyFlagMessage = `\nInteractive Transactions are not supported in the Data Proxy (${chalk.green(
+  'dataProxy',
+)} feature flag).
+Skip the ${chalk.red('interactiveTransactions')} flag:
+
+${chalk.bold(
+  highlightDatamodel(`generator client {
+    provider = "prisma-client-js"
+    previewFeatures = ["dataProxy"]
+}`),
+)}
+`
+
+// TODO
```

Comment:
Todo?

Reply:
It's a draft PR - we're still agreeing on the communication, there will be a link. Update coming soon.